tests/unit/components/Dashboard/search.spec.ts

```diff
@@ -1,16 +1,96 @@
-import { shallowMount } from '@vue/test-utils'
+import { createLocalVue, mount, shallowMount } from '@vue/test-utils'
+import { headerSearch, updatedHeaderSearch } from '../../test-data/mock-search-headers'
+
 import { Search } from '@/components/Dashboard'
+import Vuetify from 'vuetify'
+import Vuex from 'vuex'
+import { routingSlip } from '../../test-data/mock-routing-slip'
 
 describe('Search.vue', () => {
-  it('SHould have h4 title', () => {
-    const wrapper = shallowMount(Search, {})
+  const localVue = createLocalVue()
+  localVue.use(Vuex)
+  const vuetify = new Vuetify({})
+  let store
+  beforeEach(() => {
+    const routingSlipModule = {
+      namespaced: true,
+      state: {
+        routingSlip: routingSlip,
+        searchRoutingSlipParams: [],
+        searchRoutingSlipResult: []
+      },
+      actions: {
+        searchRoutingSlip: jest.fn()
+      },
+      mutations: {
+        setChequePayment: jest.fn(),
+        setCashPayment: jest.fn(),
+        setSearchRoutingSlipParams: jest.fn()
+      }
+    }
+
+    const codesModule = {
+      namespaced: true,
+      state: {
+        routingSlipStatusList: []
+      },
+      actions: {
+        getRoutingSlipStatusList: jest.fn()
+      }
+    }
+
+    store = new Vuex.Store({
+      strict: false,
+      modules: {
+        routingSlip: routingSlipModule,
+        codes: codesModule
+      }
+    })
+
+    jest.resetModules()
+    jest.clearAllMocks()
+  })
+  it('Should have h4 title', () => {
+    const wrapper = shallowMount(Search, {
+      store,
+      localVue,
+      vuetify,
+      directives: {
+        can () { /* stub */ }
+      }
+    })
     expect(wrapper.find('h4').exists()).toBeTruthy()
-    expect(wrapper.find('h4').text()).toBe('Search for Routing Slip')
+    expect(wrapper.find('h4').text()).toBe('Recent Routing Slip')
```

Comment:
How is this test working after word change ?

Reply:
refresh issue